app/webpacker/components/Panel/pages/BannedCompetitorsPage/index.jsx

```diff
@@ -39,7 +39,7 @@ export default function BannedCompetitorsPage() {
     return <Errored />;
   }
 
-  const canEditBannedCompetitors = loggedInUserPermissions.canEditGroup(bannedGroup.id);
+  const canEditBannedCompetitors = loggedInUserPermissions.canEditGroup(bannedGroup[0].id);
```

Comment:
Where does the array index zero come from? Why not 1 or 2 or 3?

Reply:
Currently there is only one banned group and it is coming as an array. Since there is only one banned group, we are taking the first banned group.

Reply:
Meh. Fine for now but not ideal for the future.